notebooks/tutorials/deploy/07-deploy-devspace.ipynb

```diff
@@ -4,8 +4,132 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "TODO"
+    "# Developing over PySyft\n",
+    "This guide is meant for developers trying to deploy PySyft locally on a Kubernetes cluster. PySyft is a powerful framework for privacy-preserving machine learning, and deploying it on Kubernetes allows an easy way to quickly try out the full PySyft stack on your own system. This guide will walk you through the process step by step. \n",
```

Comment:
I think privacy-preserving ML is only a dimension of what one could do with PySyft - to keep the messaging uniform, I'd either extend it as `privacy-preserving data analysis and machine learning` or `conducting privacy-preserving research on non-public data`